romancal/jump/jump_step.py

```diff
@@ -129,4 +129,9 @@ def process(self, input):
 
         result.meta.cal_step.jump = 'COMPLETE'
 
+        if self.save_results:
+            try:
+                self.suffix = 'jump'
+            except AttributeError:
+                self['suffix'] = 'jump'
```

Comment:
Is the try except needed, if the latter could just be used straight up?

Reply:
This is to avoid the error 
TypeError: 'DQInitStep' object does not support item assignment
when running the step from the python command line.